### DISCUSSION TITLE
Should we branch off 'next'?

### DISCUSSION THREAD

**Discussion by trentlarson** [decentralized-identity/veramo#482] [Documentation]
[CONTRIBUTING.md](https://github.com/uport-project/veramo/blob/next/CONTRIBUTING.md) says to branch off master.

- If that's the case, why does it seem that master is out-of-date, and next has more recent changes?
- If that's not the case, I'll submit a PR for to fix that instruction.

**Comment by mirceanis** [comment]
Thanks for spotting that!
Contributing.md is a bit out of date.

We use the `main` branch to represent official builds. These are released under the `@latest` channel on npmjs using semantic release logic after PRs get merged to `main`
We use the `next` branch to represent changes that are going in the next major or feature release. These are released under the `@next` channel on npmjs after PRs get merged to 'next'.

Generally it's best to branch off of `next` for general development and branch off of `main` if you want to provide a hotfix.
